src/project/internal/projectfilescontroller.cpp

```diff
@@ -486,9 +487,11 @@ io::path ProjectFilesController::selectScoreOpeningFile()
            << QObject::tr("MuseScore Dev Files") + " (*.mscs)"
            << QObject::tr("MuseScore Backup Files") + " (*.mscz~)";
 
-    return interactive()->selectOpeningFile(qtrc("project", "Score"), "", filter.join(";;"));
+    return interactive()->selectOpeningFile(qtrc("project", "Score"), ProjectConfiguration().userProjectsPath() , filter.join(";;"));
```

Comment:
Sorry, I suggested the wrong name, which has caused confusion. I meant just `configuration` instead of `projectConfiguration`. 

What you're doing now, is creating a new instance of the ProjectConfiguration class. Instead you'd want to access the existing instance through the `configuration()` method (this method is generated by one of the `INJECT` macros in projectfilescontroller.h).